### DISCUSSION TITLE
Drilling through relationships

### DISCUSSION THREAD

**Discussion by leonlazic** [Budibase/budibase#12884] [Help]
How do I drill through relationships. I know the dataprovider has a relationship field if nested but I am struggling to make it work.

I have two databases and a connecting one. They are connected through one to many relationships.

I would like to display the data in a nice custom made card from containers but I can only get either one of the databases working but not both at the same time. 

![Screenshot](https://github.com/Budibase/budibase/assets/26551422/23eb05ea-81fb-482e-9c0a-837b5df9dd47)
![Screenshot](https://github.com/Budibase/budibase/assets/26551422/8207b1ba-679b-421b-b848-35cf5461c097)
![Screenshot](https://github.com/Budibase/budibase/assets/26551422/4077969a-93c4-4964-b1cb-6af061192245)

I got the pictures working (pulled from articulator database), I also got the submit form working (update mode to used_articulator database), but I can't get the fields to display the correct values that it should pull out from the used_articulator database.
![Screenshot](https://github.com/Budibase/budibase/assets/26551422/d3603eed-4528-4b54-bb4c-0631af704508)

**Comment by ConorWebb96** [comment]
Hey @leonlazic,

You can drill into relationships with `.0` to get the first object inside the relationship and then select the field you wish to pull out. Please note the only fields that are normally available in these instances are _id and primaryDisplay.
<img width="1381" alt="Screenshot 2024-01-30 at 08 22 55" src="https://github.com/Budibase/budibase/assets/126772285/8bc3b6d1-85da-4294-978a-1908269b178f">

As you require more information than what I said is provided above. You may need to pass the _id to a [fetch row](https://docs.budibase.com/docs/data-actions#fetch-row) action to get all the information contained within that relationship. This can then be binded up to your cards with [action changing](https://docs.budibase.com/docs/chaining-actions) to save to a state variable to then be used in your cards.

**Comment by leonlazic** [comment]
So it can't be done with nested dataproviders and Links as the in the datafield dropdown?

Also why is the .0 handlebar function so limited? Why can't {{ Repeater Block 1.Employees.Jobs.0.photo.0.url }} work? If for example the Jobs database would have a "photo" column?
